src/autocomplete/EmojiProvider.tsx

```diff
@@ -62,6 +67,19 @@ function score(query, space) {
     }
 }
 
+function delimiterTrimmed(string: string): string {
```

Comment:
because our variable names get questionable when they're the same name as the type:
```suggestion
function delimiterTrimmed(str: string): string {
```

Reply:
Done.